src/constructor.js

```diff
@@ -29,8 +29,6 @@ function prepareForDestruction(self, opts) {
 
   function onDestroyed() {
     ctor.emit('destroyed', name);
-    //so we don't have to sift through all dbnames
-    ctor.emit(name, 'destroyed');
```

Comment:
This is an untested and undocumented feature that breaks if you try to emit a string like `'__proto__'`. I'm not aware of any code that is using it either in core or in plugin-land, so I propose we just remove it.